### PR TITLE
Improved landing page

### PR DESCRIPTION
**Pull request by DanielOaks**
This basically gives us an improved landing page, spelling out who we are and why implementing IRCv3 specifications is a good idea.

It splits up the current homepage into two pages, one aimed towards more generally giving an overview and an introduction to who we are, and the Working Group page that links to resources for those who are already in the WG or want more information about us.

I think this change is good, and as we go further in the future and continue to develop various parts of the site the pages will differ more from each other.

An online version of this change can be found [here](http://danieloaks.net/ircv3-site/)

---

Landing page

Landing page

---

Working Group page

ircv3 - welcome 7

### CODE REVIEW

**Comment by attilamolnar** [comment]
Is the text supposed to be final?

**Comment by DanielOaks** [comment]
Tried my best to write up something that sounds decent. If you've got suggestions or comments, definitely feel free to make them!

File: _includes/homeslider.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+</div>
+
+<div id="home-slider-wrapper"><div class="container">
+      <div>
+            <h2 class="center-text">IRCv3 Features</h2>
+
+            <ul class="homepage-features">
+                  <li>Standardised account login using SASL to speed up registration and authentication. <a href="http://ircv3.net/specs/extensions/sasl-3.1.html"><sup>[3.1]</sup></a> <a href="http://ircv3.net/specs/extensions/sasl-3.2.html"><sup>[3.2]</sup></a></li>
+                  <li>Providing the account information of other clients for the development of more advanced client features. <a href="http://ircv3.net/specs/extensions/account-notify-3.1.html"><sup>[1]</sup></a> <a href="http://ircv3.net/specs/extensions/account-tag-3.2.html"><sup>[2]</sup></a> <a href="http://ircv3.net/specs/extensions/extended-join-3.1.html"><sup>[3]</sup></a></li>
```
Comment (attilamolnar) [inline _includes/homeslider.html]: Idea: could mention away state (presence) information here as well

File: _includes/homeslider.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                  <li>Grouping related messages to simplify the display of those messages to users. <a href="http://ircv3.net/specs/extensions/batch-3.2.html"><sup>[link]</sup></a></li>
+            </ul>
+      </div>
+      <div>
+            <h2 class="center-text">What We're Working On</h2>
+
+            <ul class="homepage-features">
+                  <li>Standardised account registration and verification, allowing clients to provide better interfaces for end users. <a href="https://github.com/ircv3/ircv3-specifications/pull/152"><sup>[link]</sup></a></li>
+                  <li>Giving clients a standardised way to recognise, access and view chat history (provided by bouncers or servers). <a href="http://ircv3.net/specs/extensions/batch/chathistory-3.3.html"><sup>[link]</sup></a></li>
+                  <li>Providing a more standardised alternative to the <span class="mono">STATS</span> command that is consistent across server software. <a href="https://github.com/ircv3/ircv3-specifications/issues/99"><sup>[link]</sup></a></li>
```
Comment (attilamolnar) [inline _includes/homeslider.html]: I don't think this should be mentioned here, we aren't working on this actively and it isn't interesting to regular users as STATS is restricted to opers. Could mention providing mechanisms to secure users (and providing best practices, when we do that) or roaming instead which I believe is interesting to more people.

File: _includes/homeslider.html (hunk 1 above)
Comment (DanielOaks) [inline _includes/homeslider.html]: Sounds good, added that

File: _includes/homeslider.html (hunk 2 above)
Comment (DanielOaks) [inline _includes/homeslider.html]: Makes sense, switched it to something that should interest regular users more.

**Comment by DanielOaks** [comment]
Added the suggested changes, newer landing page:

Landing Page

**Comment by jwheare** [comment]
Only real gripe is with the word "teleconferencing", otherwise, looks good.

Perhaps:

> IRC is a networked protocol for group chat, relied upon for over 20 years.

**Comment by DanielOaks** [comment]
Centered the lists as pointed out by @dequis, modified the tagline and fixed an old issue regarding the highlighting of the navbar up the top.

Landing Page

**Comment by DanielOaks** [comment]
Alright, so this seems fine to merge? I'll have a call for last comments here

**Comment by dequis** [comment]
I'm still not very convinced by the huge green section, but none of the alternatives i suggested seem to be better (zebras distract), i'm fine with the huge green section.

http://danieloaks.net/ircv3-site/wg.html seems to be almost the same thing as the main page with slightly different text and slightly different links. I vaguely see the purpose of that section, but if anything it shouldn't include homeslider.html. Or maybe just nuke the section completely and replace working group link in index to participation?

**Comment by DanielOaks** [comment]
That page is, yes. Most of the content (ie, the big green block) on that Working Group page specifically will be replaced with more appropriate content in further PRs, really aiming it more towards people interested in joining/watching the WG activities.

With this PR, my plan is to improve the landing page to a point that's good. I'd like that Working Group page to be workable, but doing a more intensive redesign of it should be the place of another PR once this is merged imo.

I'd be fine with removing the green section from the WG page.

**Comment by jwheare** [comment]
Happy to merge now and revisit once we've discussed the roadmap (ircv3/ircv3-specifications#241)

**Comment by DanielOaks** [comment]
Merged into master

**Pull request closed without merging**